webapi/src/test/scala/org/knora/webapi/it/v2/StandoffRouteV2ITSpec.scala

```diff
@@ -233,6 +231,7 @@ class StandoffRouteV2ITSpec extends ITKnoraLiveSpec with AuthenticationV2JsonPro
       val mappingResponse         = createMapping(pathToFreetestCustomMapping, "FreetestCustomMapping")
       val mappingResponseDocument = responseToJsonLDDocument(mappingResponse)
       mappingResponse.status should equal(StatusCodes.OK)
+      // does this make sense? or how this is created? if uses projectIRI as prefix to create standoff iri then all need to be changed
```

Comment:
this should probably be clarified, rather than be a comment in code that no one will ever look at again